docs/content/release_notes/limitations.md

```diff
@@ -71,4 +76,4 @@ The following IBM block storage CSI driver features are not supported on volumes
  This limitation is tracked in the following places:
 
 - Red Hat Bug [2022328](https://bugzilla.redhat.com/show_bug.cgi?id=2022328)
-- Kubernetes Issue [106710](https://github.com/kubernetes/kubernetes/issues/106710)
\ No newline at end of file
+- Kubernetes Issue [106710](https://github.com/kubernetes/kubernetes/issues/106710)
```

Comment:
what changed here?

Reply:
I think just spacing at the endof the file

Reply:
can this be reverted? sounds like irrelevant noise for this PR

Reply:
i'm actually not sure what this issue is. i think we should just resolve it. i think it's because we added in and changed placement of content for the stretched limitations

Reply:
I don't think it's related to the rest of the file.
could you try to remove 1 new line at the end of the file?
if github won't like it, then we'll keep this unrelated change

Reply:
done